misc/services/mobile-webui/mobile-webui-frontend/src/containers/activities/distribution/DistributionStepScreen.jsx

```diff
@@ -130,7 +130,7 @@ class DistributionStepScreen extends Component {
         </div>
 
         <div className="mt-0">
-          <button className="button is-outlined complete-btn" onClick={this.handleScanButtonClick}>
+          <button className="button is-outlined complete-btn" onClick={() => this.handleScanButtonClick(true)}>
```

Comment:
what does the `locator` = true param means?

Reply:
Added a comment in the click handler